ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecFactory.java

```diff
@@ -185,6 +185,17 @@ public ProtocolSpecBuilder parisDefinition(final GenesisConfigOptions genesisCon
         evmConfiguration);
   }
 
+  public ProtocolSpecBuilder eip1153Definition(final GenesisConfigOptions genesisConfigOptions) {
```

Comment:
This should be gated by a CLI feature flag until the EIP is committed for a hard fork.  (This was how EIP-1559 and the merge were handled).  Attempts to configure it w/o the flag should produce a fatal error.